files/autopause/autopause-fcns.sh

```diff
@@ -17,26 +17,11 @@ rcon_client_exists() {
 }
 
 mc_server_listening() {
-  [[ -n $(netstat -tln | grep -e "0.0.0.0:$SERVER_PORT" -e ":::$SERVER_PORT" | grep LISTEN) ]]
+  mc-monitor status --host localhost --port $SERVER_PORT >& /dev/null
 }
 
 java_clients_connected() {
   local connections
-  connections=$(netstat -tn | grep ":$SERVER_PORT" | grep ESTABLISHED)
-  if [[ -z "$connections" ]] ; then
-    return 1
-  fi
-  IFS=$'\n'
-  connections=($connections)
-  unset IFS
-  # check that at least one external address is not localhost
-  # remember, that the host network mode does not work with autopause because of the knockd utility
-  for (( i=0; i<${#connections[@]}; i++ ))
-  do
-    if [[ ! $(echo "${connections[$i]}" | awk '{print $5}') =~ ^localhost$|^127(?:\.[0-9]+){0,2}\.[0-9]+$|^(?:0*\:)*?:?0*1$ ]] ; then
-      # not localhost
-      return 0
-    fi
-  done
-  return 1
+  connections=$(mc-monitor status --host localhost --port $SERVER_PORT --show-player-count)
```

Comment:
Prevent timeout waiting:
```suggestion
  if java_running ; then
    connections=$(mc-monitor status --host localhost --port $SERVER_PORT --show-player-count)
  else
    connections=0
  fi
```

Reply:
I actually would prefer to remove the java process check entirely. Would removing the timeout on status help then?

Reply:
Basically the same as above regarding timeout